src/Feliz.Generator.MaterialUI/ApiParser.fs

```diff
@@ -614,6 +620,9 @@ let parseProp componentMethodName (row: ComponentApiPage.Props.Row) (rowHtml: Ht
     | _, _, "any" ->
         [RegularPropOverload.create "(value: 'a)" "value"]
 
+    | _, _, "func" ->
+        [RegularPropOverload.create "(func: unit -> 'a)" "func"]
+
```

Comment:
This seems fairly specific (I haven't seen this signature many places); I suggest removing this and adding a more specific overload for a particular component/prop.